### DISCUSSION TITLE
Error In My Profile

### DISCUSSION THREAD

**Discussion by Tynab** [community/community#31253] [Other Feature Feedback, Questions, & Ideas] [answered]
I need some help. My profile doesn't show in the middle but it's on the left side :(

**Comment by KasperiP** [comment, marked as answer]
This is issue on GitHub right now and you are not the only one seeing this. According to [other discussion](https://github.com/orgs/community/discussions/31242) opened about this same topic it's already reported: https://github.com/orgs/community/discussions/31242#discussioncomment-3502858

**Reply by Tynab** [reply]
Thank you. :(

**Comment by dikkyryan** [comment]
what kind of error?

**Reply by Tynab** [reply]
github fixed it
